torch/export/unflatten.py

```diff
@@ -594,9 +594,13 @@ def _compute_accessor(parent_fqn: str, child_fqn: str) -> str:
     parent_split = parent_fqn.split(".")
     child_split = child_fqn.split(".")
 
-    assert (
-        child_split[: len(parent_split)] == parent_split
-    ), f"Child module '{child_fqn}' is not a descendant of parent module '{parent_fqn}'"
+    # TODO: support skip connection by inlining the child module.
+    if child_split[: len(parent_split)] != parent_split:
+        raise RuntimeError(
+            "Child module '%s' is not a descendant of parent mldule '%s'."
```

Comment:
This has a lot of typos and the string placeholders aren't filled in for some reason? Why did we remove the fstrings?

Reply:
Oops, it's by accident. Will fix that